wagl/acquisition/__init__.py

```diff
@@ -352,6 +404,82 @@ def acquisitions_s2_sinergise(pathname):
                                  granules=granule_groups)
 
 
+def mtl_xml_via_safe(archive, pathname):
```

Comment:
If purely related to the SAFE format by ESA, drop the "mtl" component of the function.  The "MTL" label is unique to the Landsat packages.